### PR TITLE
Update `Bleach` to version `4.1.0`

### PR DESCRIPTION
**Pull request by varlackc**
`bleach` version `4.1.0`
1. check the upstream
    https://github.com/mozilla/bleach/tree/v4.1.0

2. check the pinnings
    https://github.com/mozilla/bleach/blob/v4.1.0/tox.ini
    https://github.com/mozilla/bleach/blob/v4.1.0/setup.py
    https://github.com/mozilla/bleach/blob/v4.1.0/setup.cfg

3. check the changelogs
    https://github.com/mozilla/bleach/blob/v4.1.0/CHANGES

4. additional research
    https://github.com/conda-forge/bleach-feedstock/issues

    There are currently no open issues mentioned in `conda-forge`

5. verify dev_url
    https://github.com/mozilla/bleach

6. verify doc_url
    https://bleach.readthedocs.io/

7. pip in the test section
8. veriy the test section
9. additional tests

In order to test the `bleach` package version `4.1.0` the folowing
command was used:
`conda build bleach-feedstock --test`
the test result was the following:
`All tests passed`

Based on the research findings and the test results we can conclude
that it is safe to update `bleach` to version `4.1.0`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
```
Comment (katietz) [inline recipe/meta.yaml:18]: thanks, just add to the "build:" section the skip for python less 3.6

**Review by katietz — CHANGES_REQUESTED** [review]

**Comment by skupr-anaconda** [comment]
It would be nice to add to `test/imports` a new line:
`- bleach._vendor`
see https://github.com/mozilla/bleach/tree/main/bleach

**Comment by skupr-anaconda** [comment]
There should be in `test/requires` for the noarch python package:
  `- python <3.10`

File: recipe/meta.yaml (hunk 1 above)
Comment (varlackc) [reply recipe/meta.yaml:18]: the skip section was updated:

https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L15

**Comment by varlackc** [comment]
> It would be nice to add to `test/imports` a new line: `- bleach._vendor` see https://github.com/mozilla/bleach/tree/main/bleach

`- bleach._vendor` was added to the recipe
https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L37

**Review by skupr-anaconda — APPROVED** [review]

**Comment by varlackc** [comment]
> There should be in `test/requires` for the noarch python package: `- python <3.10`

`- python <3.10` was added to the recipe
https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L32

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,10 +29,12 @@ requirements:
 test:
   requires:
     - pip
+    - python <3.10
   commands:
     - pip check
   imports:
     - bleach
+    - bleach._vendor
```
Comment (chenghlee) [inline recipe/meta.yaml]: Packages starting with an underscore are "private" and should not be part of the import tests.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
 source:
   url: https://pypi.io/packages/source/b/bleach/bleach-{{ version }}.tar.gz
-  sha256: ffa9221c6ac29399cc50fcc33473366edd0cf8d5e2cbbbb63296dc327fb67cc8
+  sha256: 0900d8b37eba61a802ee40ac0061f8c2b5dee29c1927dd1d233e075ebf5a71da
 
 build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vvv
+  skip: true  # [py<36]
```
Comment (chenghlee) [inline recipe/meta.yaml]: Skips should not be necessary for `noarch` Python packages.

**Review by chenghlee — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (varlackc) [reply recipe/meta.yaml]: The issue has been resolved:
https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L35&L37

File: recipe/meta.yaml (hunk 3 above)
Comment (varlackc) [reply recipe/meta.yaml]: I am not sure how do we want to approach the skip section, I though that we wanted to specify the python versions in this particular case:

https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L15

**Review by varlackc — COMMENTED** [review]
Per Cheng's request the `skip` section was removed

https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L14&L16

**Review by chenghlee — APPROVED** [review]

**Review by chenghlee — APPROVED** [review]

**Pull request merged**